spec/models/numismatics/issues_feature_spec.rb

```diff
@@ -346,28 +346,13 @@
       visit edit_numismatics_issue_path(parent)
 
       doc = Nokogiri::HTML(page.body)
-      expect(doc.xpath("//issue-monograms")).not_to be_empty
-
-      issue_monogram_elements = doc.xpath("//issue-monograms")
-      expect(issue_monogram_elements).not_to be_empty
-
-      issue_monogram_element = issue_monogram_elements.first
-      attributes = JSON.parse(issue_monogram_element[":members"])
-      expect(attributes).to be_a Array
-      expect(attributes.length).to eq(2)
-
-      first_attribute = attributes.first
-      expect(first_attribute.keys).to include("id", "url", "thumbnail", "title", "attached")
-      expect(first_attribute["id"]).to eq(monogram2.id.to_s)
-      expect(first_attribute["title"]).to eq("Test Monogram")
-      expect(first_attribute["attached"]).to be true
```

Comment:
The test used to ensure that one was attached and one wasn't. Is that not important?

Reply:
Probably is. Fixed.